packages/oui-action-menu/README.md

```diff
@@ -96,6 +109,7 @@
 | `align`           | string          | @?      | yes              | `start`, `center`, `end`  | `center`   | menu alignment
 | `compact`         | boolean         | <?      | yes              | `true`, `false`           | `false`    | use the compact button
 | `disabled`        | boolean         | <?      | no               | `true`, `false`           | `false`    | disable flag
+| `placement`      | boolean    | @?        | no                | See [Popper placements](https://popper.js.org/popper-documentation.html#Popper.placements) | `bottom`           | modifier for alignment   
```

Comment:
Should be the `align` row that need to be updated (for the v2) ;)

Reply:
Oops, forgot to update that one. @AxelPeter done :)